brotli.go

```diff
@@ -8,7 +8,7 @@ import (
 
 	"github.com/andybalholm/brotli"
 	"github.com/valyala/bytebufferpool"
-	"github.com/valyala/fasthttp/stackless"
+	"github.com/laoshanxi/fasthttp/stackless"
```

Comment:
Dependence is changed by you, you should not use your own.

Reply:
thanks for remind, the second commit automatic come to the pull request, now it's fine.

Reply:
A good tip is to never make a pull request from master. Always create a new branch to use for the pull request.

Reply:
> A good tip is to never make a pull request from master. Always create a new branch to use for the pull request.

You're right, Thanks!